examples/src/multi-file/endpoints/find-users.ts

```diff
@@ -0,0 +1,19 @@
+import { api, endpoint, Optional, queryParam } from "@zenclabs/spot";
+
+@api()
+export class Api {
+  @endpoint({
+    method: "GET",
+    path: "/users"
+  })
+  getUsers(
```

Comment:
Do you mind renaming to `findUsers`? Same in the other example.

Reply:
changing it